iocage/cli/start.py

```diff
@@ -106,8 +106,3 @@ def start_jails(jails, logger, print_function):
 
     if len(failed_jails) > 0:
         exit(1)
-
-    if len(changed_jails) == 0:
-        jails_input = " ".join(list(jails))
-        logger.error(f"No jails matched your input: {jails_input}")
```

Comment:
Human users do not recognize the exit code. We need to print an error message on each failure (exit 1).

Reply:
why do you consider it a failure?

Reply:
You need to input a selector. If no jails match to this, your input is invalid and you want to know about it. Not?

Reply:
no, i really don't :P
`ioc start --rc` has a very narrow usecase
if you do it more than once, you (human) will notice a difference
if an automatism does it over and over and over again (because we have no `status`), we really don't care about "errors"

Reply:
I see two ways we could:

1.  Silence the output on autostart
2. Duplicate the loop in the autostart function and simplify the code